packages/sources/cache.gold/test/integration/lockedGold.test.ts

```diff
@@ -0,0 +1,30 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { assertError, assertSuccess } from '@chainlink/ea-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { makeExecute } from '../../src/adapter'
+
+describe('execute', () => {
+  const jobID = '1'
+  const execute = makeExecute()
+
+  describe('successful calls @integration', () => {
+    const requests = [
+      {
+        name: 'empty data',
+        testData: { data: {} },
+      },
```

Comment:
Since this is no different than the one in `packages/sources/cache.gold/test/integration/adapter.test.ts` (and won't change for other endpoints as well), does it make sense to remove it here?

Reply:
yes! removed!